container-engine-lib/lib/backend_interface/objects/service/service_config.go

```diff
@@ -93,3 +103,11 @@ func (serviceConfig *ServiceConfig) GetMemoryAllocationMegabytes() uint64 {
 func (serviceConfig *ServiceConfig) GetPrivateIPAddrPlaceholder() string {
 	return serviceConfig.privateIPAddrPlaceholder
 }
+
+func (serviceConfig *ServiceConfig) GetMinCPUAllocationMillicpus() uint64 {
```

Comment:
Can we add a comment that those are only used in the K8S backend.

Reply:
sounds good.